platform/api/aws/api.go

```diff
@@ -95,3 +97,14 @@ func (a *API) PreflightCheck() error {
 
 	return err
 }
+
+// Get the AWS partition for this region
+func (a *API) getPartition() (string, error) {
```

Comment:
We can nix this partition code.

See https://github.com/euank/mantle/commit/9ae22da24c7b0aa10ae2cd01c527ffa43c857a37

Bump the aws-sdk-go and things can suddenly look prettier :)